components/support/nimbus-fml/src/parser.rs

```diff
@@ -112,7 +114,11 @@ pub(crate) struct ImportBlock {
 #[serde(deny_unknown_fields)]
 pub(crate) struct FeatureBody {
     description: String,
-    variables: HashMap<String, FieldBody>,
+    // We need these in a deterministic order, so they are stable across multiple
+    // runs of the same manifests:
+    // 1. Swift insists on args in the same order they were declared.
+    // 2. imported features are declared and constructed in different runs of the tool.
+    variables: BTreeMap<String, FieldBody>,
```

Comment:
Does this struct map through UniFFI to Swift? If so, what type do you get out of the BTreeMap on the other side? I'm recalling problems with dictionaries in Swift being non-deterministically ordered, but that may not be a problem here.

Reply:
No, none of this code runs on device, and the code it generates is all consuming JSON objects coming through the FFI— i.e. that code doesn't correspond to anything uniffi.

Reply:
Right, that's what I expected. I was just recalling some interesting bugs around Swift dictionaries.  Carry on :)